cargo-audit/src/binary_deps.rs

```diff
@@ -27,10 +27,11 @@ pub fn load_deps_from_binary(binary_path: &Path) -> rustsec::Result<(BinaryForma
 
     use auditable_info::Error::*; // otherwise rustfmt makes the matches multiline and unreadable
     match stuff {
-        Ok(json_struct) => Ok((
-            format,
-            BinaryReport::Complete(Lockfile::try_from(&json_struct)?),
-        )),
+        // TODO(tarcieri): this is no longer supported by `auditable-serde`
+        Ok(_json_struct) => todo!("parse audit info"), // Ok((
+        //     format,
+        //     BinaryReport::Complete(Lockfile::try_from(&json_struct)?),
+        // )),
```

Comment:
I was unclear what to do here now that the `TryFrom` impl is no longer supported:

```
error[E0277]: the trait bound `Lockfile: TryFrom<&auditable_serde::VersionInfo>` is not satisfied
  --> cargo-audit/src/binary_deps.rs:32:36
   |
32 |             BinaryReport::Complete(Lockfile::try_from(&json_struct)?),
   |                                    ^^^^^^^^ the trait `From<&auditable_serde::VersionInfo>` is not implemented for `Lockfile`, which is required by `Lockfile: TryFrom<&auditable_serde::VersionInfo>`
   |
   = help: the trait `TryFrom<cargo_lock::lockfile::encoding::EncodableLockfile>` is implemented for `Lockfile`
   = help: for that trait implementation, expected `cargo_lock::lockfile::encoding::EncodableLockfile`, found `&auditable_serde::VersionInfo`
   = note: required for `&auditable_serde::VersionInfo` to implement `Into<Lockfile>`
   = note: required for `Lockfile` to implement `TryFrom<&auditable_serde::VersionInfo>`
```

Reply:
Per discussion on https://github.com/rust-secure-code/cargo-auditable/pull/160 I have vendored the relevant code